builtin/providers/aws/provider.go

```diff
@@ -215,6 +215,9 @@ func Provider() terraform.ResourceProvider {
 			"aws_lambda_permission":                        resourceAwsLambdaPermission(),
 			"aws_launch_configuration":                     resourceAwsLaunchConfiguration(),
 			"aws_lb_cookie_stickiness_policy":              resourceAwsLBCookieStickinessPolicy(),
+			"aws_elb_load_balancer_policy":                 resourceAwsLoadBalancerPolicy(),
```

Comment:
Names were attempt to align more directly with the resources as defined in the AWS APIs with the understanding that there is overlap with the `aws_lb_cookie_stickiness_policy` resource as ultimately these resources can be used to implement a load balancer cookie stickiness policy.

If anything, removing the `_elb_` interlude in the names would be more reflective of the _actual_ underlying resource names as far as the AWS API is concerned.